src/Tribe/Views/V2/Template/Excerpt.php

```diff
@@ -74,7 +74,7 @@ public function maybe_filter_excerpt_more( $link ) {
 			return $link;
 		}
 
```

Comment:
@bordoni we already discussed this here https://github.com/moderntribe/the-events-calendar/pull/2741#discussion_r333855348 and it's not possible to do that.

Reply:
Oh yeah that is true, I forgot about that, I feel like what we could do to prevent that Fatal/Infinite loop by removing the filter right after including it.

Something along the lines of:

```
		remove_filter( 'excerpt_more', [ $this, 'filter_excerpt_more' ], 999 );
```

Note, also we should move away from `999` as the priority and use something like `100`.